source/resource/visu_config.xsd

```diff
@@ -2530,4 +2530,30 @@
       </xsd:annotation>
     </xsd:attribute>
   </xsd:complexType>
+  
+  
+  <!-- OWM PlugIn -->
+  <xsd:complexType name="openweathermap">
+    <xsd:attribute name="lang" use="optional">
+      <xsd:simpleType>
+        <xsd:restriction base="xsd:string">
+          <xsd:enumeration value="en" />
+          <xsd:enumeration value="de" />
+          <xsd:enumeration value="fr" />
+        </xsd:restriction>
+      </xsd:simpleType>
+    </xsd:attribute>
+    <xsd:attribute name="appid" type="xsd:string" use="required" />
+    <xsd:attribute name="description" type="xsd:string" use="required" />
```

Comment:
Braucht man hier wirklich eine description, oder ist die nicht doch auch eher optional?

Reply:
Da kann man einen Freitext vergeben, welcher oben drüber steht. Bei description="" zeigt es die Stadt an und bei description="false" würde die Beschreibung ausgeblendet. Beschreibe ich dann in der Doku...

Reply:
wenn description leer bleiben kann, kann man das auch weglassen und damit ist es nicht required sondern optional.